project3/src/Pages/Editpost.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import { useLocation, useParams } from 'react-router-dom';
+import { useNavigate } from 'react-router';
+import axios from 'axios';
+import './style.css';
+
+export default function EditPost() {
+    const loc = useLocation();
+    let post = loc.state.post;
+    let config = loc.state.config;
+
+    const {id} = useParams();
+    const navigate = useNavigate();
+
+    let [title, setTitle] = useState(post.title);
+    let [content, setContent] = useState(post.content);
+    let [status, setStatus] = useState(post.published === true ? "Published" : "Drafted");
```

Comment:
Create constants for statuses & avoid magic strings

Reply:
Actually, user can edit post and can change status from drafted to published and vice versa, therefore kept it let

Reply:
Actually, user can edit post and can change status from drafted to published and vice versa, therefore kept it let